index.html

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>YOUR NAME HERE</title>
+  <title>Scott Madrid</title>
 </head>
 <body>
 
+<img http://giphy.com/gifs/worldsfunniestfails-fails-funniest-worlds-AxVvjFddTPBueQ1Jo4
```

Comment:
Please indent two spaces.  Also the `<img>` tag is incomplete.

Change to the following:
```html
  <img src="http://giphy.com/gifs/worldsfunniestfails-fails-funniest-worlds-AxVvjFddTPBueQ1Jo4">
```